webclient/src/views/404/i18n-404.yaml

```
@@ -1 +1,13 @@
-view_404: {}
+view_404:
+  header:
+    de: "Die angeforderte Website steht nicht zur verfügung"
```

```suggestion
    de: "Die angeforderte Seite wurde nicht gefunden."
```
I think it's a bit clearer this way ("steht nicht zur Verfügung" might mean it exists but was held back for some reason).